app/services/users/archiver.rb

```diff
@@ -10,14 +10,12 @@ class Archiver
     def archive!
       raise ArgumentError, "User already archived" if user.archived?
 
-      ApplicationRecord.transaction do
```

Comment:
This Archiver was previously called by the separation migration code - but this is no longer the case,
it is only called by `Users::MergeAndArchive` now, which wraps it in its own transaction.